prow/plugins/approve/approve_test.go

```diff
@@ -942,8 +937,7 @@ Approvers can cancel approval by writing ` + "`/approve cancel`" + ` in a commen
 			expectedComment: `[APPROVALNOTIFIER] This PR is **NOT APPROVED**
 
 This pull-request has been approved by:
-To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **cjwagner** after the PR has been reviewed.
-You can assign the PR to them by writing ` + "`/assign @cjwagner`" + ` in a comment when ready.
+To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask an approval from **cjwagner** after the PR has been reviewed.
```

Comment:
Is there a test case that cjwagner is not assigned on this PR yet, and the comment still suggests `/assign`?

Reply:
like prow/plugins/approve/approvers/approvers_test.go:866 ?

Reply:
Yeah that would work. More specifically:
-Without- the line containing `/approve cancel`
-With- the line containing `"someone else", "github.ReviewStateApproved` or `"someone else", "/approve"

Also you can test when there is only self-approval